dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1703,283 +1703,6 @@
       },
       "description": "Represents the confidence scores across all sentiment classes: positive and negative."
     },
-    "EntityList": {
```

Comment:
The problem with the list is we update categories more often than we update the api version.  If our model detects new entities, we are not able to add them to the list. Thats is why we normally keep entities as strings and not enums. PII list is set in stone so it wont be updated as often.

Reply:
I think not updating the list as frequently is fine. This will help our customers have an idea about what some values to expect. The SDKs make sure to type those entities as strings though so no issues there.

Reply:
Reverted!